python-311/README.md

```diff
@@ -237,30 +234,20 @@ See [Reference the Active Traceback Consistently](https://realpython.com/python3
 
 You can use `math.cbrt()` to calculate cube roots:
 
-```pycon
->>> import math
->>> math.cbrt(729)
-9.000000000000002
-
->>> 729**(1/3)
-8.999999999999998
-
->>> math.pow(729, 1/3)
-8.999999999999998
+```console
+(venv) $ python cube_root.py 
+math.cbrt(729) = 9.000000000000002
```

Comment:
I'm still getting 9.0 for this, even after updating to the rc:

```
$ python --version
Python 3.11.0rc1
$ python cube_root.py
math.cbrt(729) = 9.0
729 ** (1 / 3) = 8.999999999999998
math.pow(729, 1 / 3) = 8.999999999999998
```

Why does it show a more precise float representation for you?

Reply:
That is really weird. You're on macOS, right? Could it somehow be system-dependent? I've just as consistently seen the full band of 0s ...